packages/flutter_driver/lib/src/find.dart

```diff
@@ -26,7 +26,7 @@ abstract class CommandWithTarget extends Command {
   }
 
   /// Deserializes the command from JSON generated by [serialize].
-  CommandWithTarget.deserialize(Map<String, String> json)
```

Comment:
You are correct. This is not JSON, but merely name/value pairs meant to be encodable as URL parameters and sent to a JSON-RPC server. The VM service API expects these to be strings.